src/renderer/bootstrap.tsx

```diff
@@ -35,6 +47,7 @@ export {
 export async function bootstrap(App: AppComponent) {
   const rootElem = document.getElementById("app");
 
+  await attachChromeDebugger();
```

Comment:
Should we not do this for production?

Reply:
There is a check for development build inside this function, on production builds it doesn't do anything.